owasp-top10-2017-apps/a1/mongection/src/public/view/register.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Mongection</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel="icon" href="../images/favicon.png">
+    <style>
+    body {
+            background-image: url('../images/bg.jpg')
+        }
+
+        p {
+            text-align: center;
+            color: white;
+        }
+
+        div {
+            text-align: center;
+            color: white;
+        }
+    </style>
+    <script>
+        const sendRequest = async () => {
+            var name = await document.getElementById("name").value;
+            var email = await document.getElementById("email").value;
+            var password = await document.getElementById("password").value;
+            
+            const request = new XMLHttpRequest();
+            request.open("POST", "http://localhost:10001/register");
+            request.setRequestHeader("Content-Type", "application/json");
+
+            request.onreadystatechange = function() {
+                if (this.readyState == 4 && this.status == 200) {
+                    var div = document.getElementById('data').innerHTML = this.responseText;
+                }
+            };
+
+            const payload = {
+                name: name,
+                email: email,
+                password: password
+            }
+
+            const payloadJSON = JSON.stringify(payload);
+
+            request.send(payloadJSON);
+        }
+    </script>
+</head>
+<body>
+    <div id="data">
+        <p>Nome: <input type="text" name="name" id="name" value="" alt="Name"></p>
```

Comment:
I just noticed that you used "Nome" instead of "Name" for the HTML Form. Do you mind changing it to English? 🙂